src/message/MessageList.js

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
-import ActionSheet from 'react-native-actionsheet'; // eslint-disable-line
+import { connectActionSheet } from '@expo/react-native-action-sheet';
 
 import styles from '../styles';
 import TaggedView from '../native/TaggedView';
 import { LoadingIndicator } from '../common';
 import MessageTyping from '../message/MessageTyping';
 import InfiniteScrollView from './InfiniteScrollView';
 import renderMessages from './renderMessages';
-import { actionSheetButtons, executeActionSheetAction } from './messageActionSheet';
+import { constructActionButtons, executeActionSheetAction } from './messageActionSheet';
 
-export default class MessageList extends React.PureComponent {
+class MessageList extends React.PureComponent {
   autoScrollToBottom = false;
 
+  constructor(props) {
+    super();
+    this.state = { actionSheetButtons: ['', ''] };
```

Comment:
Think if you can implement this feature without using state?
I am not sure how I'll try to help with ideas.